libs/pyodide-sandbox-js/main.ts

```diff
@@ -84,7 +93,11 @@ async def install_imports(
             import micropip  # noqa
 
         for entry in to_install:
-            await micropip.install(entry["package"])
+            try:
```

Comment:
nit - weird indents, would be nice to indent properly just to be safe

Reply:
ah good catch i'll fix this up